src/libs/ReportUtils.js

```diff
@@ -1542,6 +1542,9 @@ function getProperSchemaForModifiedExpenseMessage(newValue, oldValue, valueName,
 /**
  * Get the report action message when expense has been modified.
  *
+ * ModifiedExpense::getNewDotComment in Web-Expensify should match this.
+ * If we change this function be sure to update the backend as well.
```

Comment:
NAB, this comment is well intentioned, but I wonder if we should take any additional steps besides this reminder.

Another thought, this kind of applies to basically any API call we have. Anytime the schema of the data passed to the frontend changes there is a risk of things breaking or being inconsistent.

Reply:
I was thinking the best next step is to actually get rid of this function and only format the message in Web-E. I can make an follow-up issue to address it.